news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+We thus, commend the immense contribution made by those who have helped and continue to help run this project, for managing such a large group of people so consistently for such a long period of time is absolutely no small feat.
+
+### Yuii-
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Yuii.jpg)](https://osu.ppy.sh/users/2935923)
+
+Responsible for building the foundations of the project and an acclaimed 2-year+ Beatmap Nominator to boot, Yuii-'s presence in the Mentorship Program has been instrumental in its creation and continued existence.
+
+### Halfslashed
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Halfslashed.jpg)](https://osu.ppy.sh/users/4598899)
+
+Very much the forward 'face' of the project, Halfslashed has been deep in the thick of the social aspects of leadership, being largely responsible for the deluge of speedmapping and other mapping contests that form the bulk of the competitive scene within the program.
+
+### Mir
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mir.jpg)](https://osu.ppy.sh/users/8688812)
+
+An effortless generalist and a former mentee of the program himself under Shiirn, Mir's presence in the project was confined to no one particular domain, but showcased his ability to coordinate and implement at a level that would later speak well in his favor for his entry into the Quality Assurance Team.
+
+### Mun
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Mun.jpg)](https://osu.ppy.sh/users/6699165)
+
+Once a former mentee himself under Shiirn, Mun's efforts have spanned across dozens of domains within the project, primarily handling moderation and evaluation related tasks pertaining to both mentors and mentees.
+
+### J1NX1337
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/J1nx1337.jpg)](https://osu.ppy.sh/users/3971179)
+
+Responsible for handling much of the busywork that often accompanies managing a project of such size and consistency, J1NX1337's efforts were instrumental to the continuation of the project, and while not all work is exactly glamorous, such things are certainly no less important towards making things happen.
+
+-------------
+
+As mentioned in previous years, we cannot really stress just how important community contribution is to the future of osu! as a whole. 
+
+Unlike many other game projects on the web, much of what drives osu! originates almost wholly from its community, meaning that individual people can have a **massive** influence in making the game awesome for everybody else.
+
+And we hope to see the trend continue well into our second decade of clicking circles, beating drums, catching fruit and smashing keys!
```

Comment:
This is just me, but having a line break and then starting with an and can potentially cause the reader to lose the subject.

Reply:
resolved